internal/database/external_services_test.go

```diff
@@ -1271,7 +1277,16 @@ VALUES ($1,'errored', now(), $2)
 		t.Fatal(err)
 	}
 
-	want = map[int64]string{extSvc1.ID: failure2, extSvc2.ID: ""}
+	want = []SyncError{
+		{
+			ServiceID: extSvc1.ID,
+			Message:   failure2,
+		},
+		{
+			ServiceID: extSvc2.ID,
+			Message:   "",
+		},
```

Comment:
We together checked earlier on Google Maps, so I do know where you live -- I'm pretty sure you don't have newlines growing on trees there, right?

```suggestion
		{ServiceID: extSvc1.ID, Message:   failure2},
		{ServiceID: extSvc2.ID, Message:   ""},
```

Reply:
When I was writing this line, I experimented with the suggested approach and it didn't look as clean (also had a strong feeling that you're gonna come and save the newlines here but was worth a shot anyway. 😛)

Reply:
Well, "clean" is subjective, so I can't debate that, but I personally think the one-line approach is cleaner (mainly because it doesn't pollute our codebase with wasteful newlines, you know)